R/aaa-utils.R

```diff
@@ -0,0 +1,3 @@
+.onLoad <- function(...) {
```

Comment:
Should we do it the rlang way? Should probably live in `R/zzz.R` then.

https://rlang.r-lib.org/reference/on_load.html?q=onload#comparison-with-onload-